src/Octoshift/GithubApi.cs

```diff
@@ -617,5 +619,16 @@ private static Mannequin BuildMannequin(JToken mannequin)
                                     : null
             };
         }
+
+        private void EnsureSuccessGraphQLResponse(JObject response)
+        {
+            if (response.TryGetValue("errors", out var jErrors) && jErrors is JArray { Count: > 0 } errors)
+            {
+                var error = (JObject)errors[0];
+                var errorType = error.TryGetValue("type", out var jType) ? (string)jType : null;
+                var errorMessage = error.TryGetValue("message", out var jMessage) ? (string)jMessage : null;
+                throw new OctoshiftCliException($"GraphQL response does not indicate success. Error Type: {errorType ?? "UNKNOWN"}, Error Message: {errorMessage ?? "UNKNOWN"}");
```

Comment:
IMO we should make the error simpler. Probably just `new OctoshiftCliException(errorMessage ?? "UNKNOWN")`  If users want all the gory details they can look in the verbose logs.

Reply:
addressed in c3064ce7d63ca11b181d2ab77bde4376b04f2472